### PR TITLE
[EDC-2483][EDC-2486] Get translated labels, handle default language

### PR DESCRIPTION
**Pull request by DanielM235**
- translated labels are now defined from i18N export files
- use configuration language as default language

Story [EDC-2375][EDC-2376]
Epic [EDC-1796]

### CODE REVIEW

File: src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ public void mouseExited(MouseEvent e) {
     private void openBrowser() {
         String url = "";
         try {
-            url = edcClient.getContextWebHelpUrl(mainKey, subKey, languageCode);
+            url = edcClient.getContextWebHelpUrl(mainKey, subKey, this.helpConfiguration.getLanguageCode());
```
Comment (greg7859) [inline src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java:88]: languageCode is unused.
Remove it.

File: src/test/java/fr/techad/edc/popover/swing/example/Main.java [hunk 2]
```diff
@@ -106,4 +112,22 @@ private static void createAndShowGUI() {
         f.pack();
         f.setVisible(true);
     }
+
+    private static JComboBox<String> createLangSelector() {
```
Comment (greg7859) [inline src/test/java/fr/techad/edc/popover/swing/example/Main.java:121]: Add this new option in the documentation.

**Review by greg7859 — CHANGES_REQUESTED** [review]
Need to clean the source and improve the documentation to explain how to integrate the multi langage support.

File: src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java (hunk 1 above)
Comment (DanielM235) [reply src/main/java/fr/techad/edc/popover/internal/swing/components/IconButtonListener.java:88]: Done

File: src/test/java/fr/techad/edc/popover/swing/example/Main.java (hunk 2 above)
Comment (DanielM235) [reply src/test/java/fr/techad/edc/popover/swing/example/Main.java:121]: Done

**Review by greg7859 — APPROVED** [review]

**Pull request merged**